From: Saurabh Sengar <ssengar@linux.microsoft.com>
Date: Sun, 11 Aug 2024 23:13:40 -0700
Subject: [PATCH] 
 mm-vmstat-defer-the-refresh_zone_stat_thresholds-after-all-cpus-bringup-fix

move vmstat_late_init_done under CONFIG_SMP to fix variable 'defined but
not used' warning

Link: https://lkml.kernel.org/r/1723443220-20623-1-git-send-email-ssengar@linux.microsoft.com
Signed-off-by: Saurabh Sengar <ssengar@linux.microsoft.com>
Cc: Mirsad Todorovac <mtodorovac69@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 mm/vmstat.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/vmstat.c b/mm/vmstat.c
index e361c2b3a8bc..f41984dc856f 100644
--- a/mm/vmstat.c
+++ b/mm/vmstat.c
@@ -31,7 +31,6 @@
 
 #include "internal.h"
 
-static int vmstat_late_init_done;
 #ifdef CONFIG_NUMA
 int sysctl_vm_numa_stat = ENABLE_NUMA_STAT;
 
@@ -1929,6 +1928,7 @@ static const struct seq_operations vmstat_op = {
 #ifdef CONFIG_SMP
 static DEFINE_PER_CPU(struct delayed_work, vmstat_work);
 int sysctl_stat_interval __read_mostly = HZ;
+static int vmstat_late_init_done;
 
 #ifdef CONFIG_PROC_FS
 static void refresh_vm_stats(struct work_struct *work)
-- 
2.46.1

